src/js/disability-benefits/actions/index.js

```diff
@@ -46,6 +46,22 @@ export function getClaims() {
   };
 }
 
+export function getFilteredClaims(open = true) {
+  return (dispatch) => {
+    makeAuthRequest('/v0/disability_claims',
```

Comment:
It looks like you're calling this function, which always hits the server, whenever the route changes (through `componentWillReceiveProps`).